projects/graph/src/graph.py

```diff
@@ -4,6 +4,54 @@
 
 
 class Graph:
-    """Represent a graph as a dictionary of vertices mapping labels to edges."""
     def __init__(self):
-        pass  # TODO
+        self.vertices = {}
+        self.id = -1
+        self.answer = []
+    def createID(self):
+        self.id += 1
+        return self.id
```

Comment:
Oh this is a cool idea! I was trying to implement something similar with Justin, but I couldn't execute. Having it a method in `Graph` is sweet. 😍